packages/web-components/fast-foundation/src/listbox/listbox.element.ts

```diff
@@ -19,6 +74,323 @@ export class ListboxElement extends Listbox {
     @attr({ converter: nullableNumberConverter })
     public size: number;
 
+    /**
+     * Updates the `ariaActiveDescendant` property when the active index changes.
+     *
+     * @param prev - the previous active index
+     * @param next - the next active index
+     *
+     * @internal
+     */
+    protected activeIndexChanged(prev: number | undefined, next: number): void {
+        this.ariaActiveDescendant = this.options[next]?.id ?? "";
```

Comment:
This is nullable and set as such in combobox and select; should we apply it consistently?